plugin/hpsa/hpsa.py

```diff
@@ -755,7 +761,7 @@ def volume_raid_create(self, name, raid_type, disks, strip_size,
             cmds.append("ss=%d" % int(strip_size / 1024))
 
         try:
-            self._sacli_exec(cmds, flag_convert=False)
```

Comment:
Ok, answered my own question by reading your notes in the individual commit for this change. Overprovisioning is actually a really important feature for many customers. There are workflows that involve deleting a volume and then recreating it with overprovisioning to have the Smart Array send TRIM to the disk, improving SSD performance...we'll want to be sure that a force option causes TRIM to be sent...